bokeh/models/annotations/geometry.py

```diff
@@ -0,0 +1,324 @@
+#-----------------------------------------------------------------------------
```

Comment:
I'm not sure about the module name `geometry.py` It doesn't seem descriptive to me. OTOH it's not user-visible so I am not going to quibble.

Reply:
I'm not sure either, but didn't find anything better so far.